.github/workflows/docs.yml

```diff
@@ -35,6 +35,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch all history for all tags and branches
```

Comment:
we need this if we install other packages that depend on weldx to get the correct version number for the local install